### ISSUE TITLE
[no-output-native] Missing native event name

### ISSUE THREAD

**Issue by DanielHuglen** [angular-eslint/angular-eslint#2230]
When writing an output named "search" for an Angular component, the linter does not flag it as a native event name. It seems like multiple events from the [HTMLInputElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/search_event) are missing from the [Event sources file](https://github.com/angular-eslint/angular-eslint/blob/a27993e259149cadabd38a62a67f66249f72fe81/packages/utils/src/eslint-plugin/get-native-event-names.ts), like "invalid" and "select", though "search" is the one I encountered an issue with. Is this intentional, or could it be added to the list?

Stackblitz reproduction: https://stackblitz.com/edit/angular-eslint-repros-gbbtqt6n?file=src%2Fmain.ts
Running `npm install` and `ng lint`, "click" is flagged as an error, while "search" is not

**Versions**

| package                         | version |
| ------------------------------- | ------- |
| `@angular-eslint/eslint-plugin` | `19.0.2` |
| `@typescript-eslint/parser`     | `8.19.1` |
| `ESLint`                        | `9.17.0` |
| `node`                          | `20.15.1` |

```sh
# `npx ng version` output:

Angular CLI: 19.0.7
Node: 20.15.1
Package Manager: pnpm 8.15.1
OS: win32 x64

Angular: 19.0.6
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.1900.7
@angular-devkit/build-angular      19.0.7
@angular-devkit/core               19.0.7
@angular-devkit/schematics         19.0.7
@angular/cdk                       19.0.5
@angular/cli                       19.0.7
@angular/material                  19.0.5
@angular/material-moment-adapter   19.0.5
@angular/youtube-player            19.0.5
@schematics/angular                19.0.7
rxjs                               7.8.1
typescript                         5.6.3
zone.js                            0.15.0
```

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.
